lib/resources/iis_app_pool.rb

```diff
@@ -89,7 +89,7 @@ def iis_app_pool
     script = <<~EOH
```

Comment:
Please add a comment describing why the -Compress flag is needed, and linking to the PR. Otherwise it's surprising, makes me thing we're supposed to be unpacking it through ZIP ofr something.